lib/umbrellio_utils/database.rb

```diff
@@ -22,20 +22,28 @@ def get_violated_constraint_name(exception)
       error.result.error_field(PG::Result::PG_DIAG_CONSTRAINT_NAME)
     end
 
-    def each_record(dataset, **options, &block)
-      primary_key = primary_key_from(dataset, **options)
+    def each_record(dataset, primary_key: nil, **options, &block)
```

Comment:
го тут тож мб сразу все опшны опишем

```suggestion
    def each_record(
      dataset,
      page_size: 1_000,
      sleep: nil,
      primary_key: nil,
      temp_table_name: nil,
      &block
    )
```

Reply:
Ты про доку? Ее в принципе не завезли в этом файле 🤔

Reply:
Короче добавил 👍